PendingReleaseNotes.md

```diff
@@ -20,8 +20,9 @@
 - PriorityClassNames can now be added to the Rook/Ceph components to influence the scheduler's pod preemption.
   - mgr/mon/osd/rbdmirror: [priority class names configuration settings](Documentation/ceph-cluster-crd.md#priority-class-names-configuration-settings)
   - filesystem: [metadata server settings](Documentation/ceph-filesystem-crd.md#metadata-server-settings)
-  - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)    
+  - rgw: [gateway settings](Documentation/ceph-object-store-crd.md#gateway-settings)
   - nfs: [samples](Documentation/ceph-nfs-crd.md#samples)
+- When the operator is upgraded, the mgr and osd won't be restarted
```

Comment:
I'm not sure we can say this is fully true. They will be restarted if the operator has changes for the deployment, even if the image doesn't change.

Reply:
Right, but this is less likely than the image change. I'll clarify in that sense, thanks!